src/app/App.tsx

```diff
@@ -15,15 +18,17 @@ const App: React.FC<RouteComponentProps> = ({
   const orderConfirmationPage = isPath(pathname, orderConfirmationUrl);
 
   return (
-    <>
-      <MetaConsumer />
-      <header>
-        <MainMenu />
-      </header>
-      <Routes />
-      {!orderConfirmationPage && <Footer />}
-      <OverlayManager />
-    </>
+    <ThemeProvider theme={theme}>
```

Comment:
Maybe lets call theme something like default-theme or basic-theme(this would include changing theme file name as well)

Reply:
👍